src/olympia/reviewers/serializers.py

```diff
@@ -82,46 +81,26 @@ def get_instance(self):
 
     def _get_commit(self, file_obj):
         """Return the pygit2 repository instance, preselect correct channel."""
+        if hasattr(self, '_commit'):
+            return self._commit
```

Comment:
If this serializer is called with another `file_obj`, that will return the wrong `commit` object. Is it ok?

Reply:
It would be nice to add a comment to explain that it is used to avoid a call to `revparse`.

Reply:
As discussed on Matrix, I don't think your first comment is an issue. I did add a comment regarding your second comment in the next commit.